sdk/src/types/block/slot/epoch.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use derive_more::{Deref, Display, From, FromStr};
+
+/// Timeline is divided into slots, and each epoch has a corresponding epoch index.
```

Comment:
Should this be
```suggestion
/// Timeline is divided into slots, and each slot has a corresponding epoch index.
```
?

Reply:
Nah the original was supposed to convey that each window of time has an index.